### ISSUE TITLE
Uers with thumbnailphoto in AD

### ISSUE THREAD

**Issue by ionut72** [Adldap2/Adldap2-Laravel#28]
When trying to authenticate a user that has thumbnail photo, authentication fails.

This might be that the thumbnailphoto field is not base64 encoded.

**Comment by stevebauman** [comment]
Are you sure you can't authenticate this specific user because of the thumbnail photo attribute? I don't see how this could affect authentication.

**Comment by ionut72** [comment]
I think the issue is in syncModelFromAdldap from AdldapAuthUserProvider class.

The thumbnailphoto field is not escaped and threfore it will throw and error when the $model is returned/saved. 

I can't get an acutal error in chrome because the response preview never renders. 

It worked after I added these lines of code:
```
if($modelField == 'photo')
                $adValue = 'data:image/jpeg;base64,' . base64_encode($adValue);
```

before 
```
$model->{$modelField} = $adValue;
```

**Comment by stevebauman** [comment]
Ah I understand now. Definitely unintended, thanks for the report! Fix will be out shortly.

**Comment by stevebauman** [comment]
Released `v1.3.5`, please run `composer update` to receive this fix. Thanks!
